### ISSUE TITLE
make in DigitalOcean terraform setup fails due to missing Makefiles

### ISSUE THREAD

**Issue by jdumars** [Capgemini/kubeform#130]
Build machine: Ubuntu 14.04 in AWS / t2.small
Terraform v0.6.16
Python 2.7.6
ansible 2.0.2.0

Making terraform modules fails:
```bash
ubuntu@terraformer:/tmp/kubeform/terraform/digitalocean$ for i in $(ls .terraform/modules/*/Makefile); do i=$(dirname $i); make -C $i; done
ls: cannot access .terraform/modules/*/Makefile: No such file or directory
ubuntu@terraformer:/tmp/kubeform/terraform/digitalocean$ find .terraform -name Makefile
```
The full directory tree:
```bash
.terraform/
└── modules
    ├── 06d0f380878b88f80a03504495519f7c -> /tmp/kubeform/terraform/certs/etcd
    ├── 40fb62837c2425977efcefe1e01680ee
    │   ├── ca
    │   │   ├── deploy.tf
    │   │   ├── main.tf
    │   │   └── README.md
    │   ├── docker
    │   │   ├── client
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   ├── daemon
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   └── README.md
    │   ├── etcd
    │   │   ├── main.tf
    │   │   └── README.md
    │   ├── kubernetes
    │   │   ├── admin
    │   │   │   ├── configure-kubectl.tf
    │   │   │   └── main.tf
    │   │   ├── kubelet
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   ├── master
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   └── README.md
    │   ├── LICENSE
    │   ├── README.md
    │   └── wercker.yml
    ├── 45bbc8b5c15f109ae08db77003a7a1e7
    │   ├── ca
    │   │   ├── deploy.tf
    │   │   ├── main.tf
    │   │   └── README.md
    │   ├── docker
    │   │   ├── client
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   ├── daemon
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   └── README.md
    │   ├── etcd
    │   │   ├── main.tf
    │   │   └── README.md
    │   ├── kubernetes
    │   │   ├── admin
    │   │   │   ├── configure-kubectl.tf
    │   │   │   └── main.tf
    │   │   ├── kubelet
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   ├── master
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   └── README.md
    │   ├── LICENSE
    │   ├── README.md
    │   └── wercker.yml
    ├── c58ae3c3fa465ad9e03d91dca6e82661
    │   ├── ca
    │   │   ├── deploy.tf
    │   │   ├── main.tf
    │   │   └── README.md
    │   ├── docker
    │   │   ├── client
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   ├── daemon
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   └── README.md
    │   ├── etcd
    │   │   ├── main.tf
    │   │   └── README.md
    │   ├── kubernetes
    │   │   ├── admin
    │   │   │   ├── configure-kubectl.tf
    │   │   │   └── main.tf
    │   │   ├── kubelet
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   ├── master
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   └── README.md
    │   ├── LICENSE
    │   ├── README.md
    │   └── wercker.yml
    ├── cef192a627d0557d71dd6ad77d6621c6
    │   ├── ca
    │   │   ├── deploy.tf
    │   │   ├── main.tf
    │   │   └── README.md
    │   ├── docker
    │   │   ├── client
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   ├── daemon
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   └── README.md
    │   ├── etcd
    │   │   ├── main.tf
    │   │   └── README.md
    │   ├── kubernetes
    │   │   ├── admin
    │   │   │   ├── configure-kubectl.tf
    │   │   │   └── main.tf
    │   │   ├── kubelet
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   ├── master
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   └── README.md
    │   ├── LICENSE
    │   ├── README.md
    │   └── wercker.yml
    ├── dcdbe444bc82157dcbb02eb2fc420658
    │   ├── ca
    │   │   ├── deploy.tf
    │   │   ├── main.tf
    │   │   └── README.md
    │   ├── docker
    │   │   ├── client
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   ├── daemon
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   └── README.md
    │   ├── etcd
    │   │   ├── main.tf
    │   │   └── README.md
    │   ├── kubernetes
    │   │   ├── admin
    │   │   │   ├── configure-kubectl.tf
    │   │   │   └── main.tf
    │   │   ├── kubelet
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   ├── master
    │   │   │   ├── deploy.tf
    │   │   │   └── main.tf
    │   │   └── README.md
    │   ├── LICENSE
    │   ├── README.md
    │   └── wercker.yml
    └── f8ba4470a7c46098419a0a2dd211923d
        ├── ca
        │   ├── deploy.tf
        │   ├── main.tf
        │   └── README.md
        ├── docker
        │   ├── client
        │   │   ├── deploy.tf
        │   │   └── main.tf
        │   ├── daemon
        │   │   ├── deploy.tf
        │   │   └── main.tf
        │   └── README.md
        ├── etcd
        │   ├── main.tf
        │   └── README.md
        ├── kubernetes
        │   ├── admin
        │   │   ├── configure-kubectl.tf
        │   │   └── main.tf
        │   ├── kubelet
        │   │   ├── deploy.tf
        │   │   └── main.tf
        │   ├── master
        │   │   ├── deploy.tf
        │   │   └── main.tf
        │   └── README.md
        ├── LICENSE
        ├── README.md
        └── wercker.yml

62 directories, 120 files
```

**Comment by enxebre** [comment]
Hey @jdumars you can just skip that step for Digitalocean as any of the terraform modules needs to run a Makefile in this case. I'll update the docs. We'll be providing a one click deploy script soon.
Thanks again for feedback!

**Comment by tayzlor** [comment]
docs fix in #131

**Comment by jdumars** [comment]
Glad to help!  Thanks for working on this.
